WTWSMS/common/traits/WtWSMS_new_traits_other.txt

```diff
@@ -59,4 +59,17 @@ legendary_leader = {
 	ai_ambition = 300
 	ai_greed = -300
 	ai_zeal = 10
-}
\ No newline at end of file
+}
+gothic_loyalist = {
+	culture = ostrogothic
```

Comment:
Would this work if I wanted only Ostrogoths to get this event or will this break the game?